R/param_target_weight.R

```diff
@@ -0,0 +1,21 @@
+#' Amount of supervision parameter
+#'
+#' For [uwot::umap()] and [embed::step_umap()], this is a weighting factor
+#' between data topology and target topology. A value of 0.0 weights entirely
+#' on data, a value of 1.0 weights entirely on target. The default of 0.5
+#' balances the weighting equally between data and target.
+#'
```

Comment:
this object should have a `@details` page too

Reply:
added in [e0db1be](https://github.com/tidymodels/dials/pull/324/commits/e0db1be81d7f3b37e204db9f6e35e384a04f9825)